go.mod

```diff
@@ -27,39 +27,34 @@ require (
 	github.com/gogo/status v1.0.3
 	github.com/golang/protobuf v1.3.1
 	github.com/golang/snappy v0.0.1
-	github.com/google/gofuzz v0.0.0-20161122191042-44d81051d367 // indirect
-	github.com/googleapis/gnostic v0.2.0 // indirect
 	github.com/gophercloud/gophercloud v0.0.0-20190307220656-fe1ba5ce12dd // indirect
 	github.com/gorilla/mux v1.6.2
 	github.com/gorilla/websocket v1.4.0 // indirect
 	github.com/grpc-ecosystem/go-grpc-middleware v1.0.1-0.20190118093823-f849b5445de4
 	github.com/hailocab/go-hostpool v0.0.0-20160125115350-e80d13ce29ed // indirect
-	github.com/hashicorp/consul v1.4.4
+	github.com/hashicorp/consul/api v1.1.0
```

Comment:
Why did this get downgraded?

Reply:
I did not see that! Let me check (also the package changed here).

Reply:
So `v1.4.4` is of the consul package. But the latest release of `consul/api` is `v1.1.0` https://github.com/hashicorp/consul/releases/tag/api%2Fv1.1.0